module.nix

```diff
@@ -83,6 +83,11 @@ in
       wants = [ "network-online.target" ];
       after = [ "network-online.target" "bitcoind.service" ];
       serviceConfig = {
+        # for now, run as root to avoid permission issues with eBPF/tracepoints
+        # at some point, figure out how to address this properly (e.g.,
+        # user-specific eBPF permissions or a SET_CAP binary accessible only by
+        # the user)
+        User = "root";
```

Comment:
No need to have a SET_CAP binary if using systemd. Just allow the service to use the capabilities. 

See https://github.com/0xB10C/nix/blob/7f520b3c93c57ea7934d2c84d2d459bc815e9c54/modules/peer-observer/default.nix#L135